frontend/src/components/App/NotificationIcon.js

```diff
@@ -1,6 +1,11 @@
 import React, { PropTypes, Component } from 'react'
 
 class NotificationIcon extends Component {
+
+  static propTypes = {
+    unreadNotificationsCount: PropTypes.number
+  }
```

Comment:
Yes, I prefer this pattern now, and it does let you the exports like you mention so that's nice. If you used it it'd be great